tests/integration/validations/factory-general-test.js

```diff
@@ -247,3 +247,35 @@ test("custom messages object", function(assert) {
   assert.equal(object.get('validations.attrs.firstName.isValidating'), false);
   assert.equal(object.get('validations.attrs.firstName.message'), 'Test error message');
 });
+
+test("debounced validations", function(assert) {
+  var done = assert.async();
+  var Validations = buildValidations({
+    firstName: validator(Validators.presence),
+    lastName: validator(Validators.presence, { debounce: 500 }),
+  });
+  var object = setupObject(this, Ember.Object.extend(Validations));
+
+  assert.equal(object.get('validations.isValid'), false, 'isValid was expected to be FALSE');
+  // TODO: I feel like initially a debounced validation should not be debounced.
+  assert.equal(object.get('validations.isValidating'), true, 'isValidating was expected to be TRUE');
```

Comment:
Right when we create the object, the `lastName` debounced promised is created which sets the `isValidating` to true